wlog/tcpjson/async_writer.go

```diff
@@ -18,44 +18,49 @@ import (
 	"io"
 	"log"
 
+	gometrics "github.com/palantir/go-metrics"
 	"github.com/palantir/pkg/metrics"
 	werror "github.com/palantir/witchcraft-go-error"
 )
 
 const (
 	// asyncWriterBufferCap is an arbitrarily high limit for the number of messages allowed to queue before writes will block.
-	asyncWriterBufferCapacity = 1000
+	asyncWriterBufferCapacity = 15000
```

Comment:
this is a singleton so the memory is relatively cheap -- increasing our previously arbitrary size to a larger one